src/coreclr/vm/threads.cpp

```diff
@@ -1124,22 +1124,15 @@ PCODE AdjustWriteBarrierIP(PCODE controlPc)
 extern "C" void *JIT_WriteBarrierEAX_Loc;
 #else
 extern "C" void *JIT_WriteBarrier_Loc;
+void *JIT_WriteBarrier_Loc = 0;
```

Comment:
I think this will fix the x64 duplicate symbol error:
```suggestion
#elif TARGET_X64
extern "C" void *JIT_WriteBarrier_Loc;
#else
extern "C" void *JIT_WriteBarrier_Loc;
void *JIT_WriteBarrier_Loc = 0;
```

Reply:
OK, Thanks.
I will update it now.